Packages/io.chainsafe.web3-unity.web3auth/Runtime/Plugins/Web3AuthWebGLModal.jslib

```diff
@@ -10,12 +10,10 @@ var Web3AuthWebGLNoModal =  {
             Module.dynCall_vi(login, [buffer]);
         };
     },
-    InitWeb3Auth: function (clientId, chainId, rpcTarget, displayName, blockExplorerUrl, ticker, tickerName, network) {
+    InitWeb3Auth: function (clientId, chainId, rpcTarget, displayName, blockExplorerUrl, ticker, tickerName, network, rememberMe) {
```

Comment:
Are we removing remember me?

Reply:
I thought that was causing the issue but it isn't. I'll maket this one a draft PR